### PR TITLE
Component updates for replacing the logobot

### PR DESCRIPTION
**Pull request by xli12**
* Add `<BoxIcon />` icon and `<BoxPlaceholder />` component and corresponding CSS style, story and test
* Add title attribute to `<Image />` as a prerequisite for updating the <Image /> component in another app (see [here](https://github.com/Ultimaker/stardust-marketplace_portal/blob/staging/src/views/developer_overview_view/PackageTile.tsx#L48))

### CODE REVIEW

File: src/components/image.tsx [hunk 1]
```diff
@@ -17,17 +17,20 @@ export interface ImageProps {
...
-    shape, src, size, id, className, alt,
+    shape, src, size, id, className, alt, title,
 }) => {
     const classes = classNames(`image image--${shape}`, className);
     return (
         <img
             id={id}
             src={src}
             alt={alt}
+            title={title}
```
Comment (matijs) [inline src/components/image.tsx]: I have checked the [prerequisite](https://github.com/Ultimaker/stardust-marketplace_portal/blob/staging/src/views/developer_overview_view/PackageTile.tsx#L48) but I do not understand why it's used there.

File: src/index.tsx [hunk 2]
```diff
@@ -99,6 +100,7 @@ export * from './components/input_fields/textarea_field';
 export * from './components/icons/icon_wrapper';
 export { default as ApprovedIcon } from './components/icons/approved_icon';
 export { default as BuildPlateIcon } from './components/icons/build_plate_icon';
+export { default as BoxIcon } from './components/icons/box_icon';
```
Comment (matijs) [inline src/index.tsx]: Not important for this PR, but if we would use a named export in the first place, we wouldn't have to export all the default exports as a named export here. Thoughts?

Comment (ImreBognarUltimaker) [reply src/index.tsx]: We've got a rule in eslint that specifically asks for using default exports.

Comment (ImreBognarUltimaker) [reply src/index.tsx]: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/prefer-default-export.md

File: src/components/image.tsx (hunk 1 above)
Comment (xli12) [reply src/components/image.tsx]: We have an existing `<Image />` component which can be reused in [here](https://github.com/Ultimaker/stardust-marketplace_portal/blob/staging/src/views/developer_overview_view/PackageTile.tsx#L48), to replace the `` tag. However in order to replace it while retain its original functionality, the `<Image />` component is missing the attribute `title`, which exist in marketplace_portal (see [here](https://github.com/Ultimaker/stardust-marketplace_portal/blob/staging/src/views/developer_overview_view/PackageTile.tsx#L52)).

File: src/stylesheets/_box_placeholder.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
+.box-placeholder {
+    display: inline-block;
```
[the remark this answers is not in the document]
Comment (xli12) [reply src/stylesheets/_box_placeholder.scss]: Removed: https://github.com/Ultimaker/react-web-components/pull/243/commits/2fe83d480d7c5eb9833b8c3fcaddfb950a6e6784

See here for the updated preview: (the box placeholder)

Screenshot 2019-09-16 at 15 15 57

[the remark this answers is not in the document]
Comment (ChrisTerBeke) [reply src/stylesheets/_box_placeholder.scss]: looks nice :)

**Comment by ChrisTerBeke** [comment]
Please remove branches after merging :)

**Pull request merged**